### PR TITLE
Set openshift_release explicitly

### PR DESCRIPTION
**Pull request by ingvagabund**
Make sure the `openshift_release` is always set.

### CODE REVIEW

File: sjb/config/test_cases/test_branch_openshift_ansible_logging.yml [hunk 1]
```diff
@@ -108,6 +110,7 @@ extensions:
                          -e deployment_type=origin  \
                          -e etcd_data_dir="${ETCD_DATA_DIR}" \
                          -e openshift_pkg_version="$( cat ./ORIGIN_PKG_VERSION )"               \
+                         -e openshift_release="$( cat ./ORIGIN_SHORT_VERSION )"               \
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: Is this an API change for the installer? Seems weird to need the version and release at the same time

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: Also a little upsetting that this "release" is not the RPM release field

Comment (ingvagabund) [reply sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: Please see https://trello.com/c/Z83qZeDi/530-1-require-openshiftrelease

Comment (ingvagabund) [reply sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: It's a good point. We could check if the `openshift_pkg_version` is set and if so, detect the `openshift_release` from it. @sdodson WDYT?

Comment (sdodson) [reply sjb/config/test_cases/test_branch_openshift_ansible_logging.yml]: Yeah, that seems fine. If they've specified pkg_version we can set release to the first two members of that.

**Comment by ingvagabund** [comment]
@stevekuznetsov @sdodson PTAL again. Based on a discussion we had on a meeting and comment https://github.com/openshift/openshift-ansible/pull/5860#issuecomment-339985704, the `openshift-release` is unconditionally required.

**Comment by stevekuznetsov** [comment]
Mostly LGTM. the bash bits to get the release look really fragile though -- I wonder if you could write utilities in the repos under test that would expose the correct value, like we did with `os::rpm`

**Comment by ingvagabund** [comment]
> Mostly LGTM. the bash bits to get the release look really fragile though -- I wonder if you could write utilities in the repos under test that would expose the correct value, like we did with ``os::rpm``

https://github.com/openshift/origin/pull/17355

**Comment by ingvagabund** [comment]
@stevekuznetsov updated PTAL

**Pull request merged**